### DISCUSSION TITLE
MCU Error t5uid1_ping unkown command

### DISCUSSION THREAD

**Discussion by blondsign** [Thinkersbluff/DGUS-Reloaded_for_CR6-DWIN-SET_Component#15] [Q&A]
Hi there I got following error message on my CR 6 SE 4.5.2 board. Can you provide me any solution? 

This is my make menuconfig 

<img width="1160" alt="Bildschirm­foto 2023-02-02 um 15 55 17" src="https://user-images.githubusercontent.com/10729351/216358726-978fe7fb-4a44-49a1-986b-36c9cb8bd4d5.png">

and this is the error in Klipper (Mainsail OS)

<img width="610" alt="Bildschirm­foto 2023-02-02 um 15 56 40" src="https://user-images.githubusercontent.com/10729351/216358991-6e255c5d-7f13-4ee8-bd5c-bf1823b243d5.png">

I hardly tried to follow your guidance.. but unfortunately I fail.. 
Thanks for your help!

**Comment by Thinkersbluff** [comment]
Answered in #16
